### PR TITLE
Create Vuln_Disc_WG_graduation_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
PR to recognize the Vuln Disc WG as "graduated"

### CODE REVIEW

**Review by sevansdell — APPROVED** [review]

**Review by sevansdell — APPROVED** [review]

File: process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## Working Group graduation application
+Vulnerability Disclosures WG
+
+### WG has met all Incubating requirements
+  *  n/a
+
+### List of regular contributors
+The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
+- [Christopher "CRob" Robinson, Intel](https://github.com/SecurityCRob)
+- [Jonathan Leitschuh, Dan Kaminsky Fellowship - HUMAN](https://github.com/)
```
Comment (steiza) [inline process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md:10]:
```suggestion
- [Jonathan Leitschuh, Dan Kaminsky Fellowship - HUMAN](https://github.com/JLLeitschuh)
```

File: process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [Madison Oliver, GitHub Security Lab](https://github.com/)
```
Comment (steiza) [inline process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md:11]:
```suggestion
- [Madison Oliver, GitHub Security Lab](https://github.com/taladrane)
```

File: process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- [David A Wheeler, LF/OSSF](https://github.com/david-a-wheeler)
+- [Randall T. Vasquez (SKF/Gentoo/Homebrew)](https://github.com/ran-dall)
+- [Adolfo García Veytia, Chainguard & OpenVEX](https://github.com/puerco)
+- [Andrew Pollock, Google & OSV](
```
Comment (steiza) [inline process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md:15]:
```suggestion
- [Andrew Pollock, Google & OSV](https://github.com/andrewpollock)
```

**Review by steiza — APPROVED** [review]

**Review by lehors — CHANGES_REQUESTED** [review]
LGTM but as indicated in our instructions, this PR should also change the status of the TI in the README, otherwise we'll had to do it in another PR.
https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#working-group-creation-or-change-of-lifecycle-stage

**Review by marcelamelara — APPROVED** [review]

File: process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- [Ixchel Ruiz, JFrog](https://github.com/ixchelruiz)
+- [Marcus Meissner (SUSE)](https://github.com/msmeissn)
+- [Nathan Menhorn, AMD](https://github.com/nathan-menhorn)
+- [Nicole Schwartz, ActiveState](https://github.com/NicoleSchwartz/CircuitSwan)
+- Oliver Chang, Google & OSV
+- [Paulo Flabiano Smorigo (Ubuntu/Canonical)](https://github.com/pfsmorigo)
+- Yotam Perkal, Rezilion
+
+
+### Governance
```
Comment (marcelamelara) [inline process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md:30]: Can you please add a link in here that points to the project governance documents (per the #296 TIs Gives+Gets and updated WG template)?

**Review by marcelamelara — CHANGES_REQUESTED** [review]
To align with the other TI graduated requirement for governance docs, please add a link.

**Comment by lehors** [comment]
> > LGTM but as indicated in our instructions, this PR should also change the status of the TI in the README, otherwise we'll had to do it in another PR. https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#working-group-creation-or-change-of-lifecycle-stage
> 
> I have an Issue in the vuln repo where we're tracking that, and yes once we're approved I'll adjust the WG readme to add the level and the level logo

I fixed the README for you.

**Review by lehors — APPROVED** [review]
LGTM

File: process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md (hunk 4 above)
Comment (SecurityCRob) [reply process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md:30]: added link to wg charter - https://github.com/ossf/wg-vulnerability-disclosures/blob/main/CHARTER.md

**Review by marcelamelara — APPROVED** [review]
LGTM thanks!

**Comment by SecurityCRob** [comment]
5 of 9 TAC members have approved this PR, merging.  Congrats on reaching "Graduated" level, Vuln WG!

**Pull request merged**